docs/how-to/how-to-add-custom-secret.md

```diff
@@ -1,23 +1,30 @@
 # Add custom secret to Prow
 
-This tutorial shows how to add a custom secret and use it in the Prow pipeline.
+This tutorial shows how to add and use a custom secret in the Prow pipeline.
 ## Procedure
 
-1. Add a secret to the Google Secret Manager service and apply the necessary permissions. Add the `secret-manager-trusted@sap-kyma-prow.iam.gserviceaccount.com` principal with the `Secret Manager Secret Accessor` role if the secret will be used only for a postsubmit or release job. If you are creating a Secret for a presubmit job, use `secret-manager-untrusted@sap-kyma-prow.iam.gserviceaccount.com` principal with the same role. If you want to use the secret in presubmit and postsubmit jobs, apply both principals.
+1. Create a new secret in the Secret Manager. The secret name should be in the following format: `<component>-<tool>-<secret-name>`. For example: `security-dashboard-github-bot-token`. The secret should contain the following fields:
```

Comment:
We should define what component and tool should represent. For me the component is a part composing a tool. With that understanding I would format name to <tool>-<component>-<secret-name>.

Additionally using a <secret-name> in the secret name format definition seems wrong. Maybe the last part should refer to something like <identity/entity>.